skyvern/forge/sdk/routes/agent_protocol.py

```
@@ -615,12 +628,77 @@ async def create_workflow_from_prompt(
 )
 async def import_workflow_from_pdf(
     file: UploadFile,
+    background_tasks: BackgroundTasks,
     current_org: Organization = Depends(org_auth_service.get_current_org),
```

Ruff (0.14.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)